tools/ncf.py

```diff
@@ -16,6 +16,8 @@
 import sys
 import os
 import codecs
+import ncf_constraints
+from pprint import pprint
```

Comment:
I never remove pprint, you always need it.
Having to add it each time you debug and removing it just after is very tiring. Moreover it costs almost nothing.